config/prow/plugins.yaml

```diff
@@ -34,10 +34,6 @@ owners:
   - kubernetes-sigs/contributor-playground
   - helm/charts
 
-# TODO(fejta): delete these in before April
-use_deprecated_2018_implicit_self_approve_default_migrate_before_july_2019: true
```

Comment:
Pretty sure that kubernetes still depends on this.

Reply:
But I think `HasSelfApproval` and `ConsideredReviewState` are still "true".  Therefore, I think it might be fine to delete it.
Or you mean we need to explicitly set `RequireSelfApproval` and `IgnoreReviewState`.

Reply:
ping~

Reply:
The k8s community is pretty sensitive to changes like this. 

If we want to change the default behavior of github approve to add the approve label (and we do) we'll need to give the community time to opt into continuing the current behavior of requiring an /approve message to do this.

Reply:
Hmm, these values don't appear to actually do anything anymore?

`UseDeprecatedReviewApprove`
`UseDeprecatedSelfApprove`